tests/samplers/test_seeded_generate.py

```diff
@@ -1,86 +0,0 @@
-# SPDX-License-Identifier: Apache-2.0
-# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
-"""Verify that seeded random sampling is deterministic.
-
-Run `pytest tests/samplers/test_seeded_generate.py`.
-"""
-import copy
-import random
-from itertools import combinations
-
-import pytest
-
-from vllm import SamplingParams
-from vllm.model_executor.utils import set_random_seed
-
-MODEL = "facebook/opt-125m"
-RANDOM_SEEDS = list(range(5))
-
-
-@pytest.fixture
-def vllm_model(vllm_runner, monkeypatch):
-    # This file relies on V0 internals.
```

Comment:
@WoosukKwon this one doesn't look V0 specific to me despite this comment... could we just change it to use V1?

Reply:
@njhill I think we have a counterpart in V1: https://github.com/vllm-project/vllm/blob/de533ab2a14192e461900a4950e2b426d99a6862/tests/v1/sample/test_sampling_params_e2e.py#L186